cishouseholds/edit.py

```diff
@@ -245,6 +245,15 @@ def update_visit_order(df: DataFrame, visit_order_column: str) -> DataFrame:
     return df
 
 
+def clean_barcode_simple(df: DataFrame, barcode_column: str):
+    """
+    clean barcode by converting to upper an removing whitespace
+    """
+    df = df.withColumn(barcode_column, F.upper(F.regexp_replace(F.col(barcode_column), " ", "")))
+    df = df.withColumn(barcode_column, F.regexp_replace(F.col(barcode_column), r"[^a-zA-Z0-9]", ""))
```

Comment:
I think whitespace would be captured by the pattern below, so this first `regexp_replace` call can be removed

Reply:
ill remove this from the other barcode cleaning function too then.